data/sql/updates/db_world/2019_04_26.sql

```diff
@@ -0,0 +1,11 @@
+/* QUEST :
```

Comment:
Could you please remove these unnecessary lines? You can include them in the commit description.